fdbserver/storageserver.actor.cpp

```diff
@@ -1071,10 +1073,6 @@ struct StorageServer : public IStorageMetricsService {
 	                                      // when the disk permits
 	NotifiedVersion oldestVersion; // See also storageVersion()
 	NotifiedVersion durableVersion; // At least this version will be readable from storage after a power failure
-	// In the event of the disk corruption, sqlite and redwood will either not recover, recover to durableVersion
-	// but be unable to read some data, or they could lose the last commit. If we lose the last commit, the storage
-	// might not be able to peek from the tlog (depending on when it sent the last pop). So this version just keeps
-	// track of the version we committed to the storage engine before we did commit durableVersion.
```

Comment:
Keep these? Seems they are not related to this PR.

Reply:
done